### PR TITLE
Use predefined cursor (hand) for link text

### PR DESCRIPTION
**Pull request by ozone10**
before - after
cursor

fix https://github.com/notepad-plus-plus/notepad-plus-plus/issues/12772

### CODE REVIEW

File: PowerEditor/src/WinControls/AboutDlg/URLCtrl.h [hunk 1]
```diff
@@ -26,17 +26,19 @@ class URLCtrl : public Window {
 	void create(HWND itemHandle, int cmd, HWND msgDest = NULL);
     void destroy();
 private:
+	HCURSOR& getCursor();
 	void action();
 protected :
     generic_string _URL;
     HFONT _hfUnderlined = nullptr;
     HCURSOR _hCursor = nullptr;
+	HCURSOR _hCursorPredefined = nullptr;
```
Comment (donho) [inline PowerEditor/src/WinControls/AboutDlg/URLCtrl.h:35]: Why don't we use system's one (ie. `::LoadCursor(nullptr, IDC_HAND);`) and remove XORMask & ANDMask ?

Comment (ozone10) [reply PowerEditor/src/WinControls/AboutDlg/URLCtrl.h:35]: Ok

File: PowerEditor/visual.net/notepadPlus.vcxproj [hunk 2]
```diff
@@ -130,7 +130,7 @@
...
-      <PreprocessorDefinitions>WIN32;_WIN32_WINNT=0x0600;_WINDOWS;_USE_64BIT_TIME_T;TIXML_USE_STL;TIXMLA_USE_STL;_CRT_NONSTDC_NO_DEPRECATE;_CRT_SECURE_NO_WARNINGS;_CRT_NON_CONFORMING_SWPRINTFS=1;_CRT_NON_CONFORMING_WCSTOK;_SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>OEMRESOURCE;WIN32;_WIN32_WINNT=0x0600;_WINDOWS;_USE_64BIT_TIME_T;TIXML_USE_STL;TIXMLA_USE_STL;_CRT_NONSTDC_NO_DEPRECATE;_CRT_SECURE_NO_WARNINGS;_CRT_NON_CONFORMING_SWPRINTFS=1;_CRT_NON_CONFORMING_WCSTOK;_SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```
Comment (ozone10) [inline PowerEditor/visual.net/notepadPlus.vcxproj:133]: @donho 
can we move Preprocessor Definitions outside to property sheet, so we don't have to add define to each builds

Comment (donho) [reply PowerEditor/visual.net/notepadPlus.vcxproj:133]: @ozone10 
Go ahead please. If you think it's better, it's OK to me.

Comment (ozone10) [reply PowerEditor/visual.net/notepadPlus.vcxproj:133]: @donho 
ok I will prepare it in another PR

**Pull request closed without merging**